NBitcoin.Altcoins/README.md

```diff
@@ -2,17 +2,18 @@
 
 Currently supported altcoins are:
 
-* Litecoin
-* Feathercoin
-* Dogecoin
-* Dash
-* BGold
-* Polis
 * BCash
-* Monacoin
-* Viacoin
+* BGold
+* BitCore
+* Dash
+* Dogecoin
+* Feathercoin
 * Groestlcoin
+* Litecoin
+* Monacoin
+* Polis
 * UFO
+* Viacoin
```

Comment:
why are you modifying the order of coins in the README?

Reply:
probably for alphabetical order?

Reply:
yes, for alphabetical order

Reply:
@NicolasDorier Is the alphabetical order acceptable for you?